src/modules/evaluateExpression/types.ts

```diff
@@ -20,7 +20,8 @@ export interface IGraphQLConnection {
 
 export interface IParameters {
   [key: string]: any
```

Comment:
Wouldn't this have the key `objects` now?

Reply:
Yes, good catch. It should be:
```
export interface IParameters {
  objects?: object[]
  pgConnection?: IConnection
  graphQLConnection?: IGraphQLConnection
}
```